presto-docs/src/main/sphinx/optimizer/statistics.rst

```diff
@@ -84,6 +84,17 @@ For example:
      NULL        | NULL      | NULL                  | NULL        |      15.0
      (5 rows)
 
+If there is no table layout for provided SELECT (eg. because it can be determined that WHERE will filter all
+of the table rows), than the SHOW STATS will return no statistic which will be represented as in example below.
+
+.. code-block:: none
+
+    presto:default> SHOW STATS FOR (SELECT * FROM nation WHERE nationkey > 999);
```

Comment:
somewhere we should also mention that using columns (instead of *) is not resulting in displaying fewer stats at this point.